apps/web/components/Gnb/GnbMenu.tsx

```diff
@@ -7,14 +7,14 @@ import { clsx } from "clsx";
 
 const NAV_ITEMS = [
   { href: "/dashboard", name: "대시보드", icon: PersonIcon },
-  { href: "/meeting-room", name: "회의실", icon: MeetingIcon },
+  { href: "/meetings", name: "회의실", icon: MeetingIcon },
   { href: "/seats", name: "좌석", icon: SeatsIcon },
   { href: "/equipment", name: "장비", icon: EquipmentIcon },
 ];
 
 const ADMIN_ITEM = [
```

Comment:
```suggestion
const ADMIN_ITEMS = [
```
정말 너무너무 사소해서 이런거 남겨도 되나 죄의식이 들고 주말내내 이거 괜히 남겼나 이불킥하면서 생각할거같은 리뷰이긴한데 이야기안하면 임종전에 생각날거같아서 조심스럽게 코멘트 남겨볼게

Reply:
수정완~